docs/api/qiskit/release-notes/1.0.mdx

```diff
@@ -587,7 +587,7 @@ This guide is meant to complement the release notes and have a targeted advice f
     qc.draw("mpl")
     ```
 
-    ![\_images/release\_notes-1.png](/images/api/qiskit/release_notes-1.png)
+    ![\_images/release\_notes-1.png](/images/api/qiskit/1.0/release_notes-1.png)
```

Comment:
This was changed in https://github.com/Qiskit/documentation/pull/1914. I didn't think to regenerate 1.0